lib/Db/MessageSnooze.php

```diff
@@ -48,8 +53,8 @@ class MessageSnooze extends Entity {
 	protected $srcMailboxId;
 
 	public function __construct() {
-		$this->addType('messageId', 'string');
-		$this->addType('snoozedUntil', 'integer');
+		$this->addType('mailboxId', 'integer');
+		$this->addType('uid', 'integer');
 		$this->addType('srcMailboxId', 'integer');
```

Comment:
```suggestion
		$this->addType('snoozedUntil', 'integer');
```

(No deletion. Just the snoozedUntil column is missing)